frontend/src/app/signUp/IdentityVerification/PersonalDetailsForm.tsx

```diff
@@ -0,0 +1,196 @@
+      data: personalDetails,
+      schema,
+    });
+    if (validation.valid) {
+      setErrors(initPersonalDetailsErrors());
+      setSubmitted(true);
+      return;
+    }
+    setErrors(validation.errors);
+    const alert = (
+      <Alert
+        type="error"
+        title="Form Errors"
+        body="Please check the form to make sure you complete all of the required fields."
+      />
+    );
+    showNotification(toast, alert);
+    setSaving(false);
+  };
+
+  if (submitted) {
+    return <QuestionsFormContainer personalDetails={personalDetails} />;
+  }
+
+  const getFormElement = (
+    field: keyof IdentityVerificationRequest,
+    label: string,
+    required: boolean,
+    preheader: string | null
+  ) => {
+    switch (field) {
+      case "state":
+        return (
+          <Select
+            label={label}
+            name="state"
+            value={personalDetails.state || ""}
+            options={stateCodes.map((c) => ({ label: c, value: c }))}
+            defaultSelect
+            onChange={onDetailChange("state")}
+            onBlur={() => {
+              validateField("state");
+            }}
+            validationStatus={getValidationStatus("state")}
+            errorMessage={errors.state}
+            required
+          />
+        );
+      case "dateOfBirth":
+        const now = moment();
+        return (
+          <>
+            <Label
+              htmlFor="dateOfBirth"
+              className="font-ui-sm text-bold margin-bottom-1"
+            >
+              Date of birth
+            </Label>
+            <span className="usa-hint">mm/dd/yyyy</span>
+            <DatePicker
+              id={field}
+              name={field}
+              onChange={(date) => {
+                if (date) {
+                  const newDate = moment(date)
+                    .hour(now.hours())
+                    .minute(now.minutes());
+                  onDetailChange("dateOfBirth")(newDate.format("MM/DD/YYYY"));
+                }
+              }}
+            />
+          </>
+        );
+      default:
+        return (
+          <Input
+            className={preheader ? "margin-top-0" : ""}
+            label={label}
+            type={"text"}
+            field={field}
+            key={field}
+            formObject={personalDetails}
+            onChange={onDetailChange}
+            errors={errors}
+            validate={validateField}
+            getValidationStatus={getValidationStatus}
+            required={required}
+          />
+        );
+    }
+  };
+
+  return (
+    <CardBackground>
+      <Card logo bodyKicker="Enter personal details">
+        <div className="margin-bottom-2">
+          <p className="font-ui-2xs text-base">
+            To create your account, you’ll need to reenter information and
+            answer a few questions to verify your identity directly with
+            Experian. SimpleReport doesn’t access identity verification details.
```

Comment:
I think Experian should be a link here (at least, it is in Figma). I'm not sure which page we should be linking to though - @vz3 @Rebecca-LM do you have thoughts on this?

Reply:
Maybe [this page](https://www.experian.com/decision-analytics/identity-proofing)? Either that or just the [Experian home page](https://www.experian.com/)?

Reply:
I think the first page is perfect, thanks!